### PR TITLE
[13.0] [MIG] account_invoice_report_due_list: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Please, @joao-p-marques  review it.
@Tecnativa TT26912

### CODE REVIEW

File: account_invoice_report_due_list/__manifest__.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# Copyright 2018 Tecnativa - Carlos Dauden
```
Comment (joao-p-marques) [inline account_invoice_report_due_list/__manifest__.py:1]: You should add your copyright here @victoralmau

File: account_invoice_report_due_list/models/account_move.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 Tecnativa - Carlos Dauden
```
Comment (joao-p-marques) [inline account_invoice_report_due_list/models/account_move.py:1]: Same here as above

File: account_invoice_report_due_list/tests/test_invoice_report_due_list.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Tecnativa - Vicent Cubells <vicent.cubells@tecnativa.com>
+# Copyright 2018 Tecnativa - Vicent Cubells
```
Comment (joao-p-marques) [inline account_invoice_report_due_list/tests/test_invoice_report_due_list.py:1]: Same as above

File: account_invoice_report_due_list/views/account_invoice_view.xml [hunk 4]
```diff
@@ -1,41 +1,21 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+<!-- Copyright 2018 Tecnativa - Carlos Dauden
```
Comment (joao-p-marques) [inline account_invoice_report_due_list/views/account_invoice_view.xml:2]: Same as above

File: account_invoice_report_due_list/views/report_invoice.xml [hunk 5]
```diff
@@ -1,12 +1,15 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+<!-- Copyright 2018 Tecnativa - Carlos Dauden
```
Comment (joao-p-marques) [inline account_invoice_report_due_list/views/report_invoice.xml:2]: Same as above

**Review by joao-p-marques — APPROVED** [review]
Just some small comments. Everything else looks good in the code :+1:

File: account_invoice_report_due_list/views/account_invoice_view.xml [hunk 6]
```diff
@@ -1,41 +1,22 @@
...
-                <field
-                    name="multi_date_due"
-                    attrs="{'invisible': [('multi_due', '=', False)]}"
-                />
-            </field>
-            <field name="date_due" position="attributes">
+            <field name="invoice_date_due" position="attributes">
                 <attribute
                     name="attrs"
                 >{'invisible': [('multi_due', '=', True)]}</attribute>
```
Comment (pedrobaeza) [inline account_invoice_report_due_list/views/account_invoice_view.xml]: This should be removed as Odoo already hides this when using payment term.

File: account_invoice_report_due_list/tests/test_invoice_report_due_list.py [hunk 7]
```diff
@@ -61,30 +62,27 @@ def setUpClass(cls):
         )
 
     def test_due_list(self):
-        invoice = self.env["account.invoice"].create(
+        invoice = self.env["account.move"].create(
```
Comment (pedrobaeza) [inline account_invoice_report_due_list/tests/test_invoice_report_due_list.py]: Better to use `Form` for avoiding not being resilient when other modules are installed.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**